ydb/core/tx/datashard/prefix_kmeans.cpp

```diff
@@ -298,26 +303,26 @@ class TPrefixKMeansScanBase: public TActor<TPrefixKMeansScanBase>, public NTable
         Driver->Touch(EScan::Final);
     }
 
-    EScan FeedUpload()
+    ui64 GetProbability()
+    {
+        return Rng.GenRand64();
+    }
```

Comment:
This code block unnecessary moved

Reply:
Now all upload-relevant method are groupped together